config/configuration.json

```diff
@@ -315,6 +315,24 @@
       "tags":"coastline"
     }
   },
+  "GenericTagCheck": {
+    "db": {
+      "taginfo": "extra/taginfo-db.db",
+      "wikidata": "extra/wikidata.db"
+    },
+    "tag": {
+      "usage.min": 100,
+      "percentage_of_key_for_popular": 10
+    },
+    "wikidata.tag_removal": ["abandoned", "deprecated", "imported", "obsolete", "rejected"],
+    "challenge": {
+      "description": "Tasks containing features with tags containing missing, conflicting, incorrect or illegal values",
```

Comment:
Please expand this description. For example:

> This challenge contains tasks that identify features with OSM Tags (https://wiki.openstreetmap.org/wiki/Tags) that contain invalid tag keys or missing, conflicting, incorrect or illegal tag values. In many tasks in this challenge the task will identify an illegal or questionable tag key or value and provide a link to the Control Wiki Page (wiki page with identification string starting with Q, following with 3 to 4 digits, e.g. https://wiki.osm.org/Item:Q1189) that is used to define that tag’s characteristics in a computer readable format. That Control Wiki Page should contain a link to the Documentation Wiki Page for that tag that is a human readable page that discusses the proper use of the tag and other useful information. 
> For some tasks in this challenge the task may indicate a tag value that is valid or widely used that is not documented in the wiki. For these cases the mapper should not make any changes to the OSM data, but should instead consider adding documentation to the wiki to validate the tag value officially. 
> For more information on the Atlas Check that is used to create this challenge please go to https://github.com/osmlab/atlas-checks/tree/dev/docs/checks/genericTagCheck.md

Reply:
Fixed in c3810a7a81523dda96ea4614be6b76da6ca108f3